crowdsourcing/viewsets/task.py

```diff
@@ -261,3 +284,37 @@ class CurrencyViewSet(viewsets.ModelViewSet):
 
     queryset = Currency.objects.all()
     serializer_class = CurrencySerializer
+
+
+class ExternalSubmit(APIView):
+    def post(self, request, *args, **kwargs):
+        identifier = request.GET.get('daemo_id', False)
```

Comment:
use query_params
and let's allow ajax calls as well request.POST wont be filled in that case use request.data